javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,236 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = arr => {
+  // Solution code for Challenge 1
+  let resultsArr = [];
+
+  arr.forEach(element => {
+
+    let sum = element + 1;
+    resultsArr.push(sum);
+
+  });
```

Comment:
Really awesome job on all of these! 👍 

For each function we can clean up the additional whitespace. For example on line 16  and 19, we can remove those spaces. This [guide](https://google.github.io/styleguide/jsguide.html#formatting-whitespace) has come to my rescue more than once :) Just some food for thought here 👍